site/_includes/primary-nav-items.html

```diff
@@ -10,6 +10,6 @@
     <li class="training-menu"><a href="/news/">News</a>
     </li>
     <li>
-        <button class="button-default button-green trigger-top-form"> <a href="https://github.com/apache/incubator-metron/releases" target="new">Download </a></button>
+        <button class="button-default button-green trigger-top-form"> <a href="https://dist.apache.org/repos/dist/release/incubator/metron/" target="new">Download </a></button>
```

Comment:
Just FYI, METRON-600 was created to capture the website changes that need to happen.  That commit was to get around an immediate and urgent issue with the download link (namely, it was pointing to github which hosted non-apache released artifacts).  METRON-600 is intended to fix the website with a proper download page (among other things).